python/ray/data/tests/test_dataset_formats.py

```diff
@@ -932,15 +932,48 @@ def test_parquet_read_parallel_meta_fetch(ray_start_regular_shared, fs, data_pat
 
 
 def test_parquet_reader_estimate_data_size(shutdown_only, tmp_path):
-    ds = ray.data.range(1000)
-    path = os.path.join(tmp_path, "test_parquet_dir")
-    os.mkdir(path)
-    ds.repartition(30).write_parquet(path)
+    tensor_output_path = os.path.join(tmp_path, "tensor")
+    ray.data.range_tensor(1000, shape=(1000,)).write_parquet(tensor_output_path)
+    ds = ray.data.read_parquet(tensor_output_path)
+    assert ds.num_blocks() > 1
+    data_size = ds.size_bytes()
+    assert (
+        data_size >= 7_000_000 and data_size <= 10_000_000
+    ), "estimated data size is out of expected bound"
+    data_size = ds.fully_executed().size_bytes()
+    assert (
+        data_size >= 7_000_000 and data_size <= 10_000_000
+    ), "actual data size is out of expected bound"
+
```

Comment:
For determinism, can we read it multiple times and assert the data_size the same?

Reply:
@jianoaix - sure, added.